modules/system/rolls/prayer-test.js

```diff
@@ -126,11 +126,11 @@ export default class PrayerTest extends TestWFRP {
   
   // @@@@@@@ Overcast functions placed in root class because it is used by both spells and prayers @@@@@@@
   async _overcast(choice) {
-    if (this.result.overcast.usage[choice].AoE)
-    {
-      return ui.notifications.error(game.i18n.localize("ERROR.PrayerAoEOvercast"))
+    if (this.result.overcast.usage[choice].AoE) {
+      ui.notifications.error(game.i18n.localize("ERROR.PrayerAoEOvercast"))
+    } else {
+      await super._overcast(choice)
```

Comment:
I don't think await `super._overcast` is necessary here

Reply:
are you sure? i think that default overcast options for prayer should be applied.

Reply:
Oh I specifically just mean the `await` which seems to be the only real change